themes/default/data/team/team/brant-soler.toml

```diff
@@ -0,0 +1,9 @@
+id = "brant-soler"
```

Comment:
The image for Brant needs to be renamed (`brant-soler` instead of `bran-soler`) to fix the broken image on the preview site.